src/ORM/Connect/Query.php

```diff
@@ -3,12 +3,18 @@
 namespace SilverStripe\ORM\Connect;
 
 use SilverStripe\Core\Convert;
-use Iterator;
 use Traversable;
 
 /**
- * Abstract query-result class. A query result provides an iterator that returns a map for each record of a query
- * result.
+ * Abstract query-result class. A query result provides an IteratorAggregate that returns a map for each record of a
```

Comment:
```suggestion
 * Abstract query-result class. A query result is an iterable that returns a map for each record of a
```
Lets be more general here. If we change the exact way the iterable functionality is provided in a future major release we're not likely to update this PHPDoc

Reply:
Done.